### PR TITLE
Exams

### PR DESCRIPTION
**Pull request by yaryna-bashchak**

### CODE REVIEW

File: Tasks/merge.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-   for (attribute_name in object_1) {
-       (
-          object_3[attribute_name] == object_1[attribute_name]
-       );
-        object_3[attribute_name] = object_1[attribute_name]
+const merge = (object1 = {}, object2 = {}) => {
+  const object3 = { ...object1 };
+  const keys2 = Object.keys(object2);
+  for (const key of keys2) {
+    if (!(key in object3))
```
Comment (tshemsedinov) [inline Tasks/merge.js:9]: Better use `includes`

File: Tasks/plane.js [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-    res.push(...const_plane(value));
-    arr[i] = res[i - 1]
-    }else{
-    arr[i] = res[j - 1]
-    res.push(value);
-    };
-  };
-  return res
-}
+const plane = array => {
```
Comment (tshemsedinov) [inline Tasks/plane.js:5]: Use round brackets even for single argumanet lambda functions

File: Tasks/merge.js (hunk 1 above)
Comment (yaryna-bashchak) [reply Tasks/merge.js]: > Better use `includes`

okay, I fixed it

File: Tasks/plane.js (hunk 2 above)
Comment (yaryna-bashchak) [reply Tasks/plane.js]: > Use round brackets even for single argumanet lambda functions

okay, I fixed it

**Pull request closed without merging**